public/index.html

```diff
@@ -5,10 +5,27 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <meta name="title" content="AsyncAPI Studio">
     <meta
       name="description"
       content="Studio for AsyncAPI specification, where you can validate, view preview documentation, and generate templates from AsyncAPI document."
     />
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="/">
```

Comment:
Ok last thing @mcturco 😅 You can push changes via github :)
```suggestion
    <meta property="og:url" content="https://studio.asyncapi.com/">
```